mmv1/products/vpcaccess/api.yaml

```diff
@@ -102,9 +102,10 @@ objects:
         min_version: beta
       - !ruby/object:Api::Type::Integer
         name: minThroughput
+        # its default in API doc is 200 but api returns 500 if it is not set
         description: |
-          Minimum throughput of the connector in Mbps. Default and min is 200.
-        default_value: 200
+          Minimum throughput of the connector in Mbps. Default and min is 500.
+        default_value: 500
```

Comment:
We need to remove the defaults in addition to setting optional + computed. It's fine to update the docstrings though.

```suggestion
```